src/main/scala/com/typesafe/sbt/SbtGit.scala

```diff
@@ -9,7 +9,11 @@ import com.typesafe.sbt.git.ReadableGit
 import com.typesafe.sbt.git.DefaultReadableGit
 
 /** This plugin has all the basic 'git' functionality for other plugins. */
-object SbtGit extends Plugin {
+object SbtGit extends AutoPlugin {
```

Comment:
Following the lead of `IvyPlugin`, I name my autoplugins `FooPlugin` like `AssemblyPlugin` and `ShPlugin`.